backend/src/utils/database.js

```
@@ -2,6 +2,15 @@ const { Sequelize } = require('sequelize');
 const { logger } = require('./logger');
 const createUserModel = require('../models/User');
 const createTransactionModel = require('../models/Transaction');
+
+let sequelize;
+
+const connectDB = async () => {
+  try {
+    sequelize = new Sequelize(process.env.DATABASE_URL, {
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)